controller/controller_server/decorators.py

```diff
@@ -0,0 +1,21 @@
+import grpc
+from decorator import decorator
+
+from controller.common.utils import set_current_thread_name
+from controller.controller_server.errors import ObjectAlreadyProcessingError
+from controller.controller_server.exception_handler import handle_exception, handle_common_exceptions
+
+
+def csi_method(error_response_type, lock_request_attribute=''):
+    @decorator
+    def handle_csi_method(server_method, servicer, request, context):

```

Comment:
```suggestion
    def handle_csi_method(controller_method, servicer, request, context):

```

Reply:
its also for `addon_server`

Reply:
so? it's part of the controller.
and to be honest, there isn't really an addons *server*, only an addons *servicer* anyway

Reply:
done